sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -1,39 +1,39 @@
-* [Add exception telemetry](https://github.com/Azure/azure-sdk-for-net/pull/26670)
-* [Storage transmission evaluator](https://github.com/Azure/azure-sdk-for-net/pull/26976)
-* [Add metrics exporter](https://github.com/Azure/azure-sdk-for-net/pull/26651)
-* [Add default storage location and initialization for traces](https://github.com/Azure/azure-sdk-for-net/pull/26494)
-* [Change AzureMonitorExporterLoggingExtensions from internal to public](https://github.com/Azure/azure-sdk-for-net/pull/26355)
-* [Support for exporting Activity exception
-  event](https://github.com/Azure/azure-sdk-for-net/pull/29676)
-* [Update SampleRate in Request/Dependency telemetry](https://github.com/Azure/azure-sdk-for-net/pull/31118)
+* Add histogram metric type support ([#27544](https://github.com/Azure/azure-sdk-for-net/pull/27544))
+* Update OTel package dependencies to 1.2.0 ([#28507](https://github.com/Azure/azure-sdk-for-net/pull/28507))
+* Add default storage initialization for logs and metrics ([#28506](https://github.com/Azure/azure-sdk-for-net/pull/28506))
+* Add disable storage switch ([#28446](https://github.com/Azure/azure-sdk-for-net/pull/28446))
+* Transmit from storage ([#26762](https://github.com/Azure/azure-sdk-for-net/pull/26762))
+* Add exception telemetry ([#26670](https://github.com/Azure/azure-sdk-for-net/pull/26670))
+* Storage transmission evaluator ([#26976](https://github.com/Azure/azure-sdk-for-net/pull/26976))
+* Add metrics exporter ([#26651](https://github.com/Azure/azure-sdk-for-net/pull/26651))
+* Add default storage location and initialization for traces ([#26494](https://github.com/Azure/azure-sdk-for-net/pull/26494)
+* Change AzureMonitorExporterLoggingExtensions from internal to public ([#26355](https://github.com/Azure/azure-sdk-for-net/pull/26355))
+* Support for exporting Activity exception
+  event ([#29676](https://github.com/Azure/azure-sdk-for-net/pull/29676))
+* Update SampleRate in Request/Dependency telemetry ([#31118](https://github.com/Azure/azure-sdk-for-net/pull/31118))
 
 ### Breaking Changes
 
-* [Request and Dependency Success criteria will now be decided based on
-  `Activity.Status`](https://github.com/Azure/azure-sdk-for-net/pull/31024)
-* [Changed `AzureMonitorTraceExporter` to internal](https://github.com/Azure/azure-sdk-for-net/pull/31067)
-* [Changed default offline storage directory from "Microsoft\ApplicationInsights" to "Microsoft\AzureMonitor"](https://github.com/Azure/azure-sdk-for-net/pull/31073).
+* Request and Dependency Success criteria will now be decided based on
+  `Activity.Status` ([#31024](https://github.com/Azure/azure-sdk-for-net/pull/31024))
+* Changed `AzureMonitorTraceExporter` to internal ([#31067](https://github.com/Azure/azure-sdk-for-net/pull/31067))
+* Changed default offline storage directory from "Microsoft\ApplicationInsights" to "Microsoft\AzureMonitor" ([#31073](https://github.com/Azure/azure-sdk-for-net/pull/31073))
```

Comment:
```suggestion
```

Reply:
This is not a breaking change as it was not released before.

Reply:
Do we want a line item to mention that Persistent Storage is now enabled by default?